devtools/maven/src/main/resources/create-extension-templates/integration-test-pom.xml

```diff
@@ -85,6 +85,9 @@
                     <plugin>
                         <groupId>org.apache.maven.plugins</groupId>
                         <artifactId>maven-failsafe-plugin</artifactId>
+                        <configuration>
+                            <useModulePath>false</useModulePath>
+                        </configuration>
```

Comment:
I mentioned in the issue that it might be better to use the property instead? My main reason is that people can add some other failsafe executions and it's easy to miss this config. WDYT?

Reply:
Ah, right. Let me update this.